docs/Getting started.md

```diff
@@ -0,0 +1,120 @@
+
+### Installation
+
+Installation of ImageJS is straight-forward. Use terminal to install the package:
+
+<Tabs>
+<TabItem value="npm" label="npm" default>
+
+```
+npm install image-js
+```
+
+</TabItem>
+<TabItem value="yarn" label="yarn">
+
+```
+yarn add image-js
+```
+
+</TabItem>
+</Tabs>
+
+### Importing an image
+
+There are two ways of importing an image to process it, depending on the way the user is operating: local import and import through the browser.
+
+#### Local import
+
+Local import is simple and only needs image's filepath.`decode` function will do the rest:
+
+```ts
+let parsedImage = decode(readFileSync(<filepath>));
+```
+
+#### Import via link
+
+Importing an image via browser is different. It goes through `fetch` function:
+
+```ts
+let image = async () => {
+await fetch(<image-link>)
+.then((data) => {
+  data.ArrayBuffer()
+  } // provides image raw data
+.then((data) => {
+  let view = new DataView(data); //allows data to be parsed
+  const parsedImage = decode(view); //parses data
+  return parsedImage;
+  })
+}
+
+image = image.grey(); // image is ready for usage
+```
+
+### Applying features
+
+Once the image is imported and processed, any `Image` class method can be applied as a feature. For example if you want to apply an [invert filter](/Features/Filters/Invert.md 'internal link on invert filter') you can use the invert method:
+
+```ts
+image = image.invert();
+```
+
+Image can visit "Features" category.
+
+### Saving an image
+
+#### Save an image locally
+
+Saving an image is an inverse process of importing an image.
+First you need to put the decoded data back into an image format. To do so use `encode` function:
+
+```ts
+image = image.encode();
+```
+
+To save image via Node.js use the `writeFileSync()`.
+
+```ts
+writeFileSync(<path to file>, <name of encoded image>);
+```
+
+If a file doesn't exist yet, it will be created.
+
+#### Display an image via browser
+
+To display an image via [DOM](https://en.wikipedia.org/wiki/Document_Object_Model 'wikipedia link on dom') you can add a few lines to your browser import.
+Use `querySelector` or `getElementFromId` to pick a place where an image will land on your page.
+
+```ts
+let placeToLandImage = document.querySelector('<place to put an image>');
+placeToLandImage.src = image.toDataUrl();
+```
+
+### What's next?
+
+Now that you know how images are loaded and saved you can deepen your understanding by going through Basics folder and see how different basic elements of ImageJS work. You can also broaden your horizons by looking at available Features.
+
+If you want to see how ImageJS works in practice I suggest you visit the Tutorials segment and see for yourself its practical applications.
+
+### System requirements
```

Comment:
We shouldn't mention "system requirements", just browser and NodeJS "support". It's ok to use the playwright versions for the browsers. Use the oldest version of Node.js that are tested. Don't mention operating systems.